frontend/src/app/auth.service.ts

```diff
@@ -45,55 +52,69 @@ export class AuthService {
-        user => {
-          this.changeUserEvent.next(user);
-          this.loginEvent.next({ success: true });
-          this.router.navigate(['']);
-        },
-        err => {
-          this.loginEvent.next({ success: false, message: err });
-        }
+      user => {
+        this.changeUserEvent.next(user);
+        this.loginEvent.next({ success: true });
+        this.router.navigate(['']);
+      },
+      err => {
+        this.loginEvent.next({ success: false, message: err });
+      }
       );
   }
 
   resetPassword(username: string) {
     this.http.post(
       this.resetPasswordUrl, { username }, { observe: 'response' })
       .subscribe(
-        response => {
-          if (response.status === 201) {
-            this.resetPasswordEvent.next({ success: true });
-          } else {
-            this.resetPasswordEvent.next(
-              {
-                success: false,
-                message: `Backend return http code other than 201: ${response.status}`
-              });
-          }
-        },
-        err => {
-          this.resetPasswordEvent.next({ success: false, message: err });
-        });
+      response => {
+        if (response.status === 201) {
+          this.resetPasswordEvent.next({ success: true });
+        } else {
+          this.resetPasswordEvent.next(
+            {
+              success: false,
+              message: `Backend return http code other than 201: ${response.status}`
+            });
+        }
+      },
+      err => {
+        this.resetPasswordEvent.next({ success: false, message: err });
+      });
   }
 
   confirmResetPassword(password: string, uidb64: string, token: string) {
     this.http.post(
       `${this.resetPasswordUrl}/${uidb64}/${token}`, { password }, { observe: 'response' })
       .subscribe(
-        response => {
-          if (response.status === 201) {
-            this.confirmResetPasswordEvent.next({ success: true });
-          } else {
-            this.confirmResetPasswordEvent.next(
-              {
-                success: false,
-                message: `Backend return http code other than 201: ${response.status}`
-              });
-          }
-        },
-        err => {
-          this.confirmResetPasswordEvent.next({ success: false, message: err });
-        });
+      response => {
+        if (response.status === 201) {
+          this.confirmResetPasswordEvent.next({ success: true });
+        } else {
+          this.confirmResetPasswordEvent.next(
+            {
+              success: false,
+              message: `Backend return http code other than 201: ${response.status}`
+            });
+        }
+      },
+      err => {
+        this.confirmResetPasswordEvent.next({ success: false, message: err });
+      });
+  }
+
+  register(email: string, password: string) {
+    return this.http.post(
+      this._registerUrl,
+      { password, email },
+      { observe: 'response' })
+  }
+
+  account_activation(uuid: string) {
```

Comment:
It's a method that actually does something, so please name it to indicate it's a method and not just a property (`activate_something?`). BTW, please use camelCase for variable names in JS.